video_frame_metadata_registry.src.html

```diff
@@ -61,6 +61,10 @@
     <td>segments</td>
     <td>[Human face segmentation](https://w3c.github.io/mediacapture-extensions/#human-face-segmentation)</td>
   </tr>
+  <tr>
+    <td>backgroundBlur</td>
+    <td>[Camera dffect status](https://github.com/markafoltz/camera-effects)</td>
```

Comment:
I think we would like a link to a W3C document if possible.

Given it has interaction with https://w3c.github.io/mediacapture-main/#def-constraint-backgroundBlur, I would go with adding a section to https://w3c.github.io/mediacapture-extensions/.

And we would link this new section here.

Reply:
Yes, please merge this bit of spec somewhere normative.

Reply:
nit:s/dffect/effect